projects/batfish-client/src/main/java/org/batfish/client/Client.java

```diff
@@ -1899,7 +1916,11 @@ private String loadQuestion(Path file, Map<String, String> bfq) {
         bfq.put(name.toLowerCase(), questionText);
         return name;
       } else {
-        throw new BatfishException("Question in file: '" + file + "' has no instance name");
+        if (questionText == null) {
+          throw new BatfishException("Question in file: '" + file + "' has no instance name");
+        } else {
+          throw new BatfishException("Question: '" + questionKey + "' has no instance name");
+        }
```

Comment:
seems like you could simplify this by remembering above where you got the question from. This `if` is pretty ugly.